bundles/org.openhab.binding.shelly/README.md

```diff
@@ -0,0 +1,437 @@
+|          |motionLED    |Switch   |r/w      |Control the motion LED: ON when motion is detected or OFF              |
+|          |charger      |Switch   |yes      |ON: charger connected, OFF: charger not connected.                     |
+|sensors   |temperature  |Number   |yes      |Temperature in °C                                                      |
+|          |humidity     |Number   |yes      |Relative humidity in %                                                 |
+|          |lux          |Number   |yes      |Brightness in Lux                                                      |
+|          |motion       |Switch   |yes      |ON: Motion detected, OFF: No motion (check also motionTimer)           |
+|          |last_update  |String   |yes      |Timestamp of the last update (values read by the binding)              |
+|          |event        |Trigger  |yes      |Trigger channel, receives JSON formatted event information             |
+|battery   |batteryLevel |Number   |yes      |Battery Level in %                                                     |
+|          |batteryAlert |Switch   |yes      |Low battery alert                                                      |
+
+
+## Full Example
+
+Note: PaperUI is recommended, if you want to use text files make sure to replace the thing id from you channel definition 
+
+Replace roller:XXXXXn with the last 6 digits of the Shelly device's MAC address.
+
+* .things
+
+/* Shelly 2.5 Roller */
+Thing shelly:shelly25-roller:XXXXX1 "Shelly 25 Roller XXXXX1" @ "Home Theater" [deviceIp="x.x.x.x", userId="", password=""]
+Thing shelly:shelly25-roller:XXXXX2 "Shelly 25 Roller XXXXX2" @ "Living Room"  [deviceIp="x.x.x.x", userId="admin", password="secret"]
+
+
+/* Shelly 2.5 Relays */
+Thing shelly:shelly25-relay:XXXXX3 "Shelly 25 Relay XXXXX3" @ "Hall Way" [deviceIp="x.x.x.x", userId="", password=""]
+Thing shelly:shelly25-relay:XXXXX4 "Shelly 25 Relay XXXXX4" @ "Dining Room" [deviceIp="x.x.x.x", userId="", password=""]
+Thing shelly:shelly25-relay:XXXXX5 "Shelly 25 Relay XXXXX5" @ "Bed Room" [deviceIp="x.x.x.x", userId="", password=""]
+
+* .items
+
+Switch Shelly_XXXXX3_Relay "Garage Light" (garage) {channel="shelly:shelly1:XXXXX3:relay#output"}
+Switch Shelly_XXXXX3_OverPower "Garage Light Over Power" (garage) {channel="shelly:shelly1:XXXXX3:relay#overpower"}
+Switch Shelly_XXXXX3_OverTemp "Garage Light Over Temperature" (garage) {channel="shelly:shelly1:XXXXX3:relay#overtemperature"}
+Number Shelly_XXXXX3_AutoOnTimer "Garage Light Auto On Timer" (garage) {channel="shelly:shelly1:XXXXX3:relay#autoOn"}
+Number Shelly_XXXXX3_AutoOffTimer "Garage Light Auto Off Timer" (garage){channel="shelly:shelly1:BA2F18:relay#autoOff"}
+Switch Shelly__TimerActive "Garage Light Timer Active" (garage) {channel="shelly:shelly1:BA2F18:relay#timerActive"}
+
+/* Power Meter */
+Number Shelly_BA2F18_Power "Bath Room Light Power" (bathroom) {channel="shelly:shelly1:BA2F18:meter#currentWatts"}
+
+
```

Comment:
Please also add a short sitemap snippet here to make the example complete.

Reply:
done